src/main/scala/is/hail/io/vcf/ExportVCF.scala

```diff
@@ -38,6 +38,23 @@ object ExportVCF {
     }
   }
 
+  def emitGenotype(f: Field, sb: StringBuilder, value: Annotation) {
+    if (value == null)
+      sb += '.'
```

Comment:
Do we output ./. for a missing genotype now?  I don't think it should make a difference...